api/mapping/views.py

```diff
@@ -300,7 +320,7 @@ def get_context_data(self, **kwargs):
 @method_decorator(login_required, name="dispatch")
 class StructuralMappingTableListView(ModelFormSetView):
```

Comment:
Please, convert this class to a ListView, and remove the approved field from the screen. This is turning into a visualisation page, instead of a form.

Reply:
will leave this for a 3rd PR 👍